test/corpus/todo.txt

```diff
@@ -74,24 +74,6 @@ end
             (self)
             (operator)))))))
 
-=======================
-misc keywords / methods
-=======================
-
-super
-previous_def
```

Comment:
These still need to be added as keywords, right?

Reply:
I didn't think it worth it as there's nothing special about their parsing and can be handled in highlighting by matching the name identifier. I can add them though if you think it's worth it

Reply:
Okay, they can just be parsed as regular calls